runtime/src/main/java/io/novafoundation/nova/runtime/multiNetwork/chain/ChainSyncService.kt

```diff
@@ -37,7 +37,7 @@ class ChainSyncService(
 
         val remoteChains = retryUntilDone { chainFetcher.getChains() }
 
-        val newChains = remoteChains.map { mapRemoteChainToLocal(it, oldChainsById[it.chainId], gson) }
+        val newChains = remoteChains.map { mapRemoteChainToLocal(it, oldChainsById[it.chainId], isCustomNetwork = false, gson) }
```

Comment:
It is not an issue in the scope of this PR, but once weimplement "add network" feature we will need to adapt chain diffing here in ChainSyncService since otherwise it will remove manually added chains that are not present in chains.json upon each sync

Reply:
Agree. I think we can handle chains here by source